### PR TITLE
max colorscale option

### LINKED ISSUE #198: cubeslice broken
```
bassler@grgalt:~/Projects/shieldhit/test/26_voxel$ cubeslice --data myvox02.dos --ct ../../res/TST000/TST000000.ctx 
Traceback (most recent call last):
  File "/home/bassler/.local/bin/cubeslice", line 11, in <module>
    sys.exit(main())
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/utils/cubeslice.py", line 158, in main
    logger.info("Cube 1 " + args.dos + " shape {:d} x {:d} x {:d}".format(data_cube.dimx, data_cube.dimy,
AttributeError: 'Namespace' object has no attribute 'dos'
bassler@grgalt:~/Projects/shieldhit/test/26_voxel$ 
```

### LINKED ISSUE #304: cubeslice should have option for setting colour scale limits
e.g. "--max=120" when reading an LET cube to cap it at 120 keV/um

wrongly reported here: https://github.com/DataMedSci/pymchelper/issues/198

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #304

### CODE REVIEW

File: pytrip/utils/cubeslice.py [hunk 1]
```diff
@@ -133,6 +133,8 @@ def main(args=sys.argv[1:]):
         "-f", "--from", type=int, dest='sstart', metavar='N', help="Output from slice number N", default=1)
     parser.add_argument("-t", "--to", type=int, dest='sstop', metavar='M', help="Output up to slice number M")
     parser.add_argument("-H", "--HUbar", dest='HUbar', default=False, action='store_true', help="Add HU colour bar")
+    parser.add_argument("-m", "--max", type=float, dest='csmax', metavar='csmax',
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: @nbassler any comments on description ?

**Comment by grzanka** [comment]
Applying option still to be implemented

File: pytrip/utils/cubeslice.py (hunk 1 above)
Comment (nbassler) [inline pytrip/utils/cubeslice.py]: is ok for now

**Comment by grzanka** [comment]
tst003001_048
tst003001_048 copy 1

What about LET above scale, should it be plotted with last color or not plotted at all ?
Above the second case

**Comment by nbassler** [comment]
It should be made in such a way, that the same value has the same color across all images, when using --max function, this is useful for visual relative comparisons.

**Comment by grzanka** [comment]
tst003001_048
tst003001_048 copy 1

The colorscale is the same for all slices, but mapping color-value is changing for different `--max` settings.

If `--max` is used then hottest color means "maximum value or more".

Should I change it that the `--max` doesn't affect color-value mapping ?

**Comment by nbassler** [comment]
hmm, i think most intuitve would be that the highest colour is mapped to --max. When stuff goes beyond it is just mapped to the same colour (--max)

**Comment by grzanka** [comment]
OK, so right now the code in this PR is written in such way that highest colour is mapped to `--max` or to maximum value of dataset (if --max not present).

Option `--max` applies only to data cube (dose, let), not to CT cubes.

Feel free to test (and squash merge if you like it)

**Pull request merged**